webcompat/views.py

```diff
@@ -245,7 +245,7 @@ def download_file(filename):
         Python app.
         '''
         return send_from_directory(
-            app.config['UPLOADS_DEFAULT_DEST'] + '/uploads', filename)
+            app.config['UPLOADS_DEFAULT_DEST'], filename)
```

Comment:
`/uploads` was added to `[UPLOADS_DEFAULT_DEST]`, because that's specific to how someone wants to set up their system.